site/html_template.js

```diff
@@ -0,0 +1,16 @@
+const htmlTemplate = `<!DOCTYPE html>
+<html lang='en'>
+<head>
+  <title><!-- Your title goes here --></title>
+  <meta charset='utf-8'>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='shortcut icon' href='/path/to/favicon' type='image/x-icon'>
```

Comment:
Should we say `href='Your favicon path goes here'` to make it more clear that this is another fill-in-the-black situation?